aiida/engine/processes/calcjobs/calcjob.py

```diff
@@ -244,6 +244,8 @@ def define(cls, spec: CalcJobProcessSpec) -> None:  # type: ignore[override]
             help='If set to true, the submission script will load the system environment variables',)
         spec.input('metadata.options.environment_variables', valid_type=dict, default=lambda: {},
             help='Set a dictionary of custom environment variables for this calculation',)
+        spec.input('metadata.options.envvar_double_quotes', valid_type=bool, default=False,
+            help='If set to true, use double quotes for the environment variables.',)
```

Comment:
```suggestion
            help='If set to True, use double quotes instead of single quotes to escape the environment variables specified in ``environment_variables``.',)
```